bot/exts/utils/attachment_pastebin_uploader.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import annotations
+
+import re
+
+import aiohttp
+import discord
+from discord.ext import commands
+from pydis_core.utils import paste_service
+
+from bot.bot import Bot
+from bot.constants import Emojis
+from bot.log import get_logger
+
+log = get_logger(__name__)
+
+PASTEBIN_UPLOAD_EMOJI = Emojis.check_mark
+DELETE_PASTE_EMOJI = Emojis.trashcan
+
+
+class EmbedFileHandler(commands.Cog):
+    """
+    Handles automatic uploading of attachments to the paste bin.
+
+    Whenever a user uploads one or more attachments that is text-based (py, txt, csv, etc.), this cog offers to upload
+    all the attachments to the paste bin automatically. The steps are as follows:
+    - The bot replies to the message containing the attachments, asking the user to react with a checkmark to consent
+        to having the content uploaded.
```

Comment:
@swfarnsworth 

I think Joe is saying that this line does not line up with the previous line, it seems one or two spaces too deeply indented (though I am viewing this on mobile so maybe it just looks wrong?